Add DALI Center integration documentation

niracler opened the pull request:
<!--
  You are amazing! Thanks for contributing to our project!
  Please, DO NOT DELETE ANY TEXT from this template! (unless instructed).

  Before submitting your pull request, please verify that you have chosen the correct target branch,
  and that the PR preview looks fine and does not include unrelated changes.
-->
## Proposed change
<!-- 
    Describe the big picture of your changes here to communicate to the
    maintainers why we should accept this pull request. If it fixes a bug
    or resolves a feature request, be sure to link to that issue in the 
    additional information section.
-->

Add comprehensive documentation for the new DALI Center integration that supports Sunricher DALI gateways. This documentation covers automatic discovery, configuration, features, troubleshooting, and technical details for the integration.

The integration allows Home Assistant users to control DALI (Digital Addressable Lighting Interface) devices through Sunricher DALI gateways with real-time updates and comprehensive light control capabilities.

## Type of change
<!--
    What types of changes does your PR introduce to our documentation/website?
    Put an `x` in the boxes that apply. You can also fill these out after
    creating the PR.
-->

- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [ ] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [x] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [x] I've opened up a PR to add logos and icons in [Brands repository](https://github.com/home-assistant/brands).
- [ ] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information
<!--
    Details are important, and help maintainers processing your PR.
    Please be sure to fill out additional details, if applicable.
-->

- Link to parent pull request in the codebase: https://github.com/home-assistant/core/pull/151479
- Link to parent pull request in the Brands repository:  https://github.com/home-assistant/brands/pull/7807
- This PR fixes or closes issue: fixes #

## Checklist
<!--
    Put an `x` in the boxes that apply. You can also fill these out after
    creating the PR. If you're unsure about any of them, don't hesitate to ask.
    We're here to help! This is simply a reminder of what we are going to look
    for before merging your code.
-->

- [x] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [x] The documentation follows the Home Assistant documentation [standards].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

niracler committed:
Add DALI Center integration documentation

Add documentation for DALI Center integration supporting Sunricher DALI gateways with automatic discovery, real-time updates, and light control features.

niracler committed:
Merge branch 'next' into dali-center-docs

niracler committed:
Merge branch 'next' into dali-center-docs

niracler committed:
Merge branch 'next' into dali-center-docs

niracler committed:
Update DALI Center integration docs

niracler committed:
Update DALI Center integration docs

niracler committed:
From dali_center to sunricher_dali_center

niracler committed:
Merge branch 'next' into dali-center-docs

niracler committed:
Add removal instructions to Sunricher DALI Center doc

niracler committed:
Merge branch 'dali-center-docs' of https://github.com/niracler/home-assistant.io into dali-center-docs

joostlek commented on source/_integrations/sunricher_dali_center.markdown:
```diff
@@ -0,0 +1,52 @@
+description: Instructions on how to integrate DALI Center with Home Assistant.
+ha_category:
+  - Light
+ha_release: 2025.10
```
```suggestion
ha_release: 2025.11
```

joostlek commented on source/_integrations/sunricher_dali_center.markdown:
```diff
@@ -0,0 +1,52 @@
+ha_integration_type: hub
+---
+
+The DALI Center integration connects Home Assistant to Sunricher SR-GW-EDA DALI gateways so you can manage DALI lighting from one place. Gateways are discovered on the local network and push updates keep entity state in sync.
```
Should we add an url?

joostlek commented on source/_integrations/sunricher_dali_center.markdown:
```diff
@@ -0,0 +1,52 @@
+## Setup
+
+Configure the integration from the Home Assistant user interface; discovery finds supported gateways automatically.
+
```
RE: discovery, in the future we could see if we could discover the device on the network via DHCP, mDNS or SSDP

niracler replied:
We’ll likely adopt SSDP for gateway discovery in the future, though some of the config flows will probably need to be redesigned.

joostlek commented on source/_integrations/sunricher_dali_center.markdown:
```diff
@@ -0,0 +1,52 @@
+
+{% warning %}
+Removing the integration removes all associated DALI devices from Home Assistant.
+{% endwarning %}
```
I think that is obvious when removing the integration

joostlek commented on source/_integrations/sunricher_dali_center.markdown:
```diff
@@ -0,0 +1,52 @@
+
+{% include integrations/config_flow.md %}
+
+After setup the gateway appears under **Devices & Services**, and Home Assistant creates light entities for every supported DALI device. Multiple gateways are supported, each with its own set of entities.
```
I think we explain the light entities in the Lights section already

niracler committed:
Update DALI Center integration docs

c0ffeeca7 commented on source/_integrations/sunricher_dali_center.markdown:
```diff
@@ -0,0 +1,48 @@
+This integration follows standard integration removal; no extra steps are required.
+
+{% include integrations/remove_device_service.md %}
+
```
```suggestion
```

c0ffeeca7 commented on source/_integrations/sunricher_dali_center.markdown:
```diff
@@ -0,0 +1,48 @@
+
+- **Gateway not discovered:** Confirm the gateway and Home Assistant share the same network, the gateway is powered, and its web interface is reachable; restart the gateway if needed.
+- **Devices missing:** Check DALI bus wiring, ensure devices are commissioned on the gateway, then reload the integration.
+- **Entities unavailable:** Verify device power and addressing in the gateway before restarting the integration.
```
```suggestion
- **Entities unavailable:** Verify device power and addressing in the gateway before restarting the integration.

## Removing the integration

This integration follows standard integration removal; no extra steps are required.

{% include integrations/remove_device_service.md %}

```

c0ffeeca7 committed:
move removal section to end of page

c0ffeeca7 commented on source/_integrations/sunricher_dali_center.markdown:
```diff
@@ -0,0 +1,49 @@
+
+{% include integrations/config_flow.md %}
+
+After setup, the gateway appears under **Devices & Services**. Multiple gateways are supported, each with its own set of entities.
```
```suggestion
After setup, the gateway appears under {% my integrations title="**Settings** > **Devices & services**" %}. Multiple gateways are supported, each with its own set of entities.
```

c0ffeeca7 committed:
add my link

c0ffeeca7 commented on source/_integrations/sunricher_dali_center.markdown:
```diff
@@ -0,0 +1,49 @@
+
+## Troubleshooting
+
+- **Gateway not discovered:** Confirm the gateway and Home Assistant share the same network, the gateway is powered, and its web interface is reachable; restart the gateway if needed.
```
You could use the Symptom/Resolution structure for the Troubleshooting section
https://github.com/home-assistant/home-assistant.io/blob/current/source/_integrations/_integration_docs_template.markdown?plain=1#L169

niracler replied:
Thanks for pointing out the template! I realized I hadn't thoroughly reviewed the [documentation standards](https://developers.home-assistant.io/docs/documenting/standards). I'll rewrite the documentation following the template tomorrow.

niracler replied:
The Troubleshooting section was removed as the setup process is now clearly documented with step-by-step instructions and a diagram, and there are no integration-specific issues that commonly occur.

niracler committed:
Improve DALI Center documentation

Follow Home Assistant documentation standards with better structure, clearer setup steps, and gateway reset diagram.

c0ffeeca7 commented on source/_integrations/sunricher_dali_center.markdown:
```diff
@@ -0,0 +1,72 @@
+- Sunricher SR-GW-EDA DALI gateway on the same network as Home Assistant
+- DALI lighting devices connected to and configured on the gateway
+
+## Setup
```
```suggestion
```
The configuration include already comes with a title.
<img width="889" height="730" alt="image" src="https://github.com/user-attachments/assets/1c41cd64-c7d0-4839-98de-e581d988c284" />

niracler committed:
Update source/_integrations/sunricher_dali_center.markdown

Co-authored-by: c0ffeeca7 <38767475+c0ffeeca7@users.noreply.github.com>

c0ffeeca7 commented on source/_integrations/sunricher_dali_center.markdown:
```diff
@@ -0,0 +1,71 @@
+2. Select **Submit** to start discovery. Home Assistant will search for gateways for up to 3 minutes.
+3. While discovery is in progress, press the **Reset** button on your DALI gateway device once. The gateway will respond immediately after the button press.
+
+![Press the Reset button on the DALI gateway](/images/integrations/sunricher_dali_center/gateway_reset_button.png)
```
```suggestion
3. While discovery is in progress, press the **Reset** button on your DALI gateway device once. The gateway will respond immediately after the button press.
   ![Press the Reset button on the DALI gateway](/images/integrations/sunricher_dali_center/gateway_reset_button.png)
```

c0ffeeca7 committed:
Update source/_integrations/sunricher_dali_center.markdown

c0ffeeca7 reviewed (approved):
Thank you, @niracler :+1:
